packages/store/addon/-private/system/record-arrays/record-array.js

```diff
@@ -9,6 +9,15 @@ import { Promise } from 'rsvp';
 import { DEBUG } from '@glimmer/env';
 import { PromiseArray } from '../promise-proxies';
 import SnapshotRecordArray from '../snapshot-record-array';
+import { internalModelFactoryFor } from '../store/internal-model-factory';
+
+function recordForIdentifier(store, identifier) {
+  return internalModelForIdentifier(store, identifier).getRecord();
+}
+
+function internalModelForIdentifier(store, identifier) {
+  return internalModelFactoryFor(store).lookup(identifier);
+}
```

Comment:
globally available utility functions or are you ok leaving them per test module?

Reply:
per test module seems fine for now, as I'd like to kill off these helpers as well in a follow up.